### DISCUSSION TITLE
Add support for MSIX and APPX windows installer

### DISCUSSION THREAD

**Discussion by ntindle** [marcelotduarte/cx_Freeze#2474] [Ideas]
With python3.13, msilib is getting removed from python. 

cx_Freeze should move to different packaging methods for windows installs

See:
https://github.com/python/cpython/issues/104773
https://learn.microsoft.com/en-us/windows/msix/package/create-app-package-with-makeappx-tool

**Comment by marcelotduarte** [comment]
https://github.com/marcelotduarte/cx_Freeze/discussions/1721#discussioncomment-4496526
